generators/client/templates/angular/package.json.ejs

```diff
@@ -157,10 +158,10 @@
     "build": "<%= clientPackageManager %> run webpack:prod",
     "test": "<%= clientPackageManager %> run lint && karma start src/test/javascript/karma.conf.js",
     "test:watch": "<%= clientPackageManager %> test -- --watch",
-    "webpack:dev": "<%= clientPackageManager %> run webpack-dev-server -- --config webpack/webpack.dev.js --progress --inline --hot --profile --port=9060 --watch-content-base",
-    "webpack:build:main": "<%= clientPackageManager %> run webpack -- --config webpack/webpack.dev.js --progress --profile",
+    "webpack:dev": "<%= clientPackageManager %> run webpack-dev-server --config webpack/webpack.dev.js --progress --inline --hot --profile --port=9060 --watch-content-base",
```

Comment:
@sendilkumarn Does this work with npm? i guess it still needs the --

Reply:
ahh.... I forgot that we added it for `npm` updated in #7199